components/eam/src/chemistry/modal_aero/seasalt_model.F90

```diff
@@ -417,47 +481,11 @@ subroutine seasalt_emis(u10, u10cubed, lchnk, srf_temp, ocnfrc, ncol, cflx, emis
 
    mass_frac_bub_section(:ncol,:,:) = 0.0_r8
    om_ssa(:ncol,:) = 0.0_r8
-   F_eff(:ncol) = 0.0_r8
 
    !if (fmoa==1) then ! Burrows et al. organic sea spray
```

Comment:
This if condition in the comment can be removed. Also, do we want to remove the author name from the name of variables and subroutines and put in the comments? I don't see much variable/subroutines named with authors unless they are related to an algorithm.

Reply:
changed the comment and removed fmoa. removed burrows in the variable names and subroutines